library/std/src/keyword_docs.rs

```diff
@@ -128,27 +130,27 @@ mod break_keyword {}
 /// const WORDS: &'static str = "hello rust!";
 /// ```
 ///
-/// Thanks to static lifetime elision, you usually don't have to explicitly use 'static:
+/// Thanks to static lifetime elision, you usually don't have to explicitly use `'static`:
```

Comment:
```suggestion
/// Thanks to static lifetime elision, you usually don't have to explicitly use `static`:
```

Reply:
Hmm, I feel like it's more clear the way it currently is. Why do you suggest removing the lifetime marker `'`?

Reply:
Oops, this is what happens when you do drive-by PRs 😆 I missed that it's a lifetime. It's fine as-is then.